src/Vatras/Data/Prop/Properties.agda

```diff
@@ -0,0 +1,134 @@
+module Vatras.Data.Prop.Properties {F : Set} where
+
+open import Data.Bool as Bool using ()
+open import Data.Bool.Properties using (∧-comm; ∧-zeroˡ; ∧-zeroʳ)
+open import Data.Empty using (⊥)
+open import Data.Product as Product using (Σ; _×_; ∃-syntax; _,_)
+open import Data.Sum as Sum using (_⊎_) renaming (inj₁ to left; inj₂ to right)
```

Comment:
I'm not a huge fan of these renames. When reading the code you always have to look at the imports or jump to the definition just to know what these names mean. What's the problem with the standard names `inj₁`, `inj₂`?

Reply:
You are right, we should use the default names to increase readability. Personally, I just like the Haskell-Style names more than inj_... but we should stick to the standard. I probably intended to remove the renaming after finishing the file anyway but forgot.